Interfaces/API/DiracAdmin.py

```diff
@@ -253,7 +253,7 @@ def getSiteMaskLogging(self, site=None, printOutput=False):
 
       sitesLogging = result['Value']
       if isinstance(sitesLogging, dict):
-        for siteName, tupleList in sitesLogging.iteritems():
+	for siteName, tupleList in sitesLogging.items():  # can be an iterator
```

Comment:
I guess this indentation should have been:
```suggestion
	  for siteName, tupleList in sitesLogging.items():  # can be an iterator
```